docs/components/movement-sensor/imu/imu-wit-hwt905.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Configure a WitMotion HWT905-TTL IMU"
+linkTitle: "imu-wit-hwt905"
+weight: 10
+type: "docs"
+description: "Configure a WitMotion HWT905-TTL IMU on your machine. Once configured, use the API to obtain the AngularVelocity, Orientation, CompassHeading and LinearAcceleration."
+images: ["/icons/components/imu.svg"]
+# SMEs: Bucket Team
+---
+
+An [inertial measurement unit (IMU)](https://en.wikipedia.org/wiki/Inertial_measurement_unit) provides data for the [`AngularVelocity`](/components/movement-sensor/#getangularvelocity), [`Orientation`](/components/movement-sensor/#getorientation), [`CompassHeading`](/components/movement-sensor/#getcompassheading), [`LinearAcceleration`](/components/movement-sensor/#getlinearacceleration), and [`GetAccuracy`](/components/movement-sensor/#getaccuracy) methods.
```

Comment:
A possibly off-topic comment: I tried clicking on these from [1], and they all link to places half a page up from where the topic actually starts, so I accidentally start reading the topic just before the one I clicked on. It's as though clicking on these tries to get the relevant section to be about a third of the way from the bottom of my browser window, rather than at the top of the window. Is that a mistake, or on purpose?

[1] https://docs-test.viam.dev/2911/components/movement-sensor/imu/imu-wit-hwt905/

Reply:
Yeah…. @npentrel has looked into this before I believe

Reply:
kk. As long as it's not a surprise. If there are changes to be made, they belong in a separate PR.